app/controllers/application_controller.rb

```diff
@@ -1,23 +1,13 @@
 class ApplicationController < ActionController::Base
   helper_method :current_user
-  helper_method :find_bookmark
   helper_method :list_tags
-  helper_method :tutorial_name
 
   add_flash_types :success
 
   def current_user
     @current_user ||= User.find(session[:user_id]) if session[:user_id]
   end
 
-  def find_bookmark(id)
```

Comment:
It looks to me like `find_bookmark` will probably be used in the completion of #11 

So I think this should probably be preserved/tested

Reply:
Discussed together that we will leave this deleted for now and if we need to rebuild later, we will. Closing.